### ISSUE TITLE
Users have to manually install extensions

### ISSUE THREAD

**Issue by cassidyjames** [ublue-os/beyond#74]
Right now, users have to manually run `just install-beyond-extensions` after logging in. This is not great! Instead, we should add them to Yafti once it supports installing GNOME extensions.

Blocked by https://github.com/ublue-os/yafti/issues/110

**Comment by tulilirockz** [comment]
we could try using the "beyond first boot" script to install these extensions - like, we could just run the custom.just's install-beyond-extensions recipe on it then by writing a "firstboot-done" on /etc/ublue-os/beyond-first-boot (or something like that) we can verify whether or not the script ran and if run accordingly

i'd just make a pull request for that right now, but I don't know if that's something that makes sense for this project..?

**Comment by cassidyjames** [comment]
What I did in #87 (because the extension was not packaged anywhere) was just included the extension in tree. This could be an option, similar to how Inter is handled, though it seems a little odd and will not benefit from updates without OS updates.

**Comment by castrojo** [comment]
I've thought about trying this, put the extensions on the disk and then let them update, but not sure how that would work long-term. I think they'd autoupdate themselves right away?

**Comment by cassidyjames** [comment]
@castrojo if you include the extension files directly in the image, they show as "system" installed extensions, and so don't get updated e.g. from the Extensions Manager app. Whatever files are included in the image is what will be run. 

So the nice side is that everyone on the same commit of the OS is running the exact same extension code, but the downside is that extension updates can only come with an OS update/reboot. I think that's... fine? You don't get a new GNOME Shell update without rebooting into a new commit of the OS, either.

**Comment by castrojo** [comment]
Yeah, that's probably better than having to remind people to run a just command.
